app/assets/src/components/views/pipeline_viz.scss

```diff
@@ -4,38 +4,62 @@
 .pipelineViz {
   display: flex;
   justify-content: center;
+  align-items: center;
 }
 
 .stage {
   display: flex;
   justify-content: center;
   align-items: center;
-  margin: 24px;
+  margin: 24px 0;
 }
 
 .stageButton {
   background-color: $light-grey;
   border-radius: 8px;
   padding: 12px;
-  width: 120px;
+  width: 250px;
```

Comment:
Can this be automatic width base on nowrap text?
Or is it supposed to have the same width for all the stages?
If the latter, can you make sure the text wraps if ever necessary `word-break: break-word`.

Reply:
The mocks have them all at the same width but I can ask jenn about it on Monday to confirm!